src/server/scripts/EasternKingdoms/ZulGurub/boss_marli.cpp

```diff
@@ -77,142 +81,187 @@ enum Misc
 float const DamageIncrease = 35.0f;
 float const DamageDecrease = 100.f / (1.f + DamageIncrease / 100.f) - 100.f;
 
+float const MaxRangeHatch = 100.f;
+
 class boss_marli : public CreatureScript
 {
     public: boss_marli() : CreatureScript("boss_marli") { }
 
         struct boss_marliAI : public BossAI
         {
-            boss_marliAI(Creature* creature) : BossAI(creature, DATA_MARLI) { }
+            public:
+                boss_marliAI(Creature* creature) : BossAI(creature, DATA_MARLI) { }
 
-            void Reset() override
-            {
-                if (events.IsInPhase(PHASE_THREE))
-                    me->ApplyStatPctModifier(UNIT_MOD_DAMAGE_MAINHAND, TOTAL_PCT, DamageDecrease); // hack
-                _Reset();
-            }
+                void Reset() override
+                {
+                    if (events.IsInPhase(PHASE_THREE))
+                        me->ApplyStatPctModifier(UNIT_MOD_DAMAGE_MAINHAND, TOTAL_PCT, DamageDecrease); // hack
+                    /*std::for_each(_hatchedEggsGuids.begin(), _hatchedEggsGuids.end(), [=](ObjectGuid guid)
+                        {
+                            if (GameObject * egg = ObjectAccessor::GetGameObject(*me, guid))
+                            {
+                                egg->Respawn();
+                            }
+                        });*/
 
-            void JustDied(Unit* /*killer*/) override
-            {
-                _JustDied();
-                Talk(SAY_DEATH);
-            }
+                    summons.DespawnAll();
+                    //_hatchedEggsGuids.clear();
+                    _Reset();
+                }
+                
+                //void SpellHitTarget(GameObject* target, SpellInfo const* /*spellInfo*/) override
+                //{
+                //    if(target->GetEntry() == GOB_SPIDER_EGG)
+                //        _hatchedEggsGuids.insert(target->GetGUID());
+                //}
 
-            void JustEngagedWith(Unit* /*who*/) override
-            {
-                _JustEngagedWith();
-                events.ScheduleEvent(EVENT_SPAWN_START_SPIDERS, 1s, 0, PHASE_ONE);
-                Talk(SAY_AGGRO);
-            }
+                void JustDied(Unit* /*killer*/) override
```

Comment:
are these just formatting changes ? it makes it hard to review the PR, is it possible to revert the formatting changes ?

Reply:
Yea I totally agree, must have clicked Ctrl + K + D accidentally leading into auto formatting, will fix this.